docs/src/app/components/pages/components/Stepper/Page.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+
+import CodeExample from '../../../CodeExample';
+import PropTypeDescription from '../../../PropTypeDescription';
+import MarkdownElement from '../../../MarkdownElement';
+
+import stepperReadmeText from './README';
+
+import VerticalLinearStepper from './VerticalLinearStepper';
+import VerticalNonLinearStepper from './VerticalNonLinearStepper';
+import VerticalLinearStepperWithOptionalStep from './VerticalLinearStepperWithOptionalStep';
+import VerticalLinearStepperCode from
+'!raw!./VerticalLinearStepper';
+import VerticalLinearStepperWithOptionalStepCode from
+'!raw!./VerticalLinearStepperWithOptionalStep';
+import VerticalNonLinearStepperCode from
+'!raw!./VerticalNonLinearStepper';
+
+import HorizontalLinearStepper from './HorizontalLinearStepper';
+import HorizontalLinearStepperCode from '!raw!./HorizontalLinearStepper';
+
+import stepperCode from '!raw!material-ui/lib/stepper/stepper';
+import stepCode from '!raw!material-ui/lib/stepper/VerticalStep';
+
+const descriptions = {
+  verticalLinearStepper: 'The vertical linear stepper requires steps be completed in specific order',
+  verticalLinearStepperWithOptionalStep: 'Set the `optional` property to true for optional steps.'
+   + 'And we should pass a custom label view through `customLabelView` property to show'
+   + ' the difference between optional step and normal step',
+  verticalNonLinearStepper: 'For the vertical non linear stepper, steps can be completed in any order',
+  horizontalLinearStepper: 'The horizontal linear stepper is the same with vertical linear stepper',
```

Comment:
Better to repeat the description, to save the user cross-referencing, but you could begin with 'As for the vertical linear stepper, ...'